src/github_services.py

```diff
@@ -76,17 +82,18 @@ def _get_request_headers() -> Dict[str, str]:
 def get_prs_assigned_to_reviewers(
     org_name: str,
     repository: str,
-    wait_hours: int
+    max_waiting_time: int
 ) -> DefaultDict[str, List[github_domain.PullRequest]]:
-    """Fetches all the PRs on the given repository and returns a list of PRs
-    assigned to reviewers.
+    """Fetches all PRs and returns a list of PRs assigned to reviewers.
 
     Args:
-        org_name: str. name of the github organization.
-        repository: str. name of the github repository.
-        wait_hours: int. The threshold to count a PR for discussion comment.
+        org_name: str. GitHub organization name.
+        repository: str. GitHub repository name.
+        max_waiting_time: int. The maximum time in hour to wait for a review. Any PR
```

Comment:
hour --> hours

Any PR exceed ... --> If the waiting time for a PR review has exceeded this limit, the corresponding reviewer should be notified.

Reply:
done